examples/machine-set.yaml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: Cluster
+metadata:
+  name: test
+  namespace: test
+spec:
+  clusterNetwork:
+    services:
+      cidrBlocks:
+        - "10.0.0.1/24"
+    pods:
+      cidrBlocks:
+        - "10.0.0.2/24"
+    serviceDomain: example.com
+  providerConfig:
+    value:
+      apiVersion: clusteroperator.openshift.io/v1alpha1
+      kind: ClusterProviderConfigSpec
+      clusterId: test
+      clusterVersionRef:
+        namespace: test
+        name: test
+      hardware:
+        aws:
+          accountSecret:
+            name: test-aws-creds
+          sshSecret:
+            name: test-ssh-key
+          sshUser: centos
+          sslSecret:
+            name: test-certs
+          region: eu-west-1
+          keyPairName: test
+      defaultHardwareSpec:
+        aws:
+          instanceType: t1.micro
+      machineSets:
+      - nodeType: Master
+        size: 1
+      - shortName: infra
+        nodeType: Compute
+        infra: true
+        size: 1
+      - shortName: compute
+        nodeType: Compute
+        size: 1
+---
```

Comment:
This is not needed.

Reply:
this seems be convention everywhere https://github.com/kubernetes/examples/blob/master/guestbook/all-in-one/guestbook-all-in-one.yaml#L17 https://github.com/kubernetes-sigs/cluster-api/blob/master/clusterctl/examples/google/addons.yaml.template#L2 and I think it makes it a bit more readable